packages/utils/testing/src/lib/index.js

```diff
@@ -7,6 +7,16 @@ const fakeEmbark = (config) => {
   const events = new Events();
   const plugins = new Plugins();
 
+  const ipc = {
```

Comment:
Could this be configurable somehow, e.g. when you want to test from the perspective of `isClient()` being `true`?

Reply:
Yes, I think in that case we want `IPC` to be injectable so users can provide their own "mock" implementation where this applies.

Reply:
In other words, instead of using the `fakeEmbark()` function that takes care of that for you with decent defaults, you'll have to instantiate `Embark()` yourself. So this already works.